MonoGame.Framework/Graphics/Texture.DirectX.cs

```diff
@@ -27,33 +28,33 @@ public IntPtr GetSharedHandle()
 
         internal abstract SharpDX.Direct3D11.Resource CreateTexture();
 
-        internal SharpDX.Direct3D11.Resource GetTexture()
+        internal virtual SharpDX.Direct3D11.Resource GetTexture()
```

Comment:
This method was made virtual, but I don't see anything that overrides it.  We try to keep the use of `virtual` to a minimum because it takes longer to call a virtual than a regular method.